src/otg/datasource/intervals/thurnman.py

```diff
@@ -19,7 +19,7 @@ class IntervalsThurnman(Intervals):
     """Interval dataset from Thurman et al. 2012."""
 
     @staticmethod
-    def read_thurnman(spark: SparkSession, path: str):
```

Comment:
Is the person `Thurman` or `Thurnman`? Not a crutial detail though. I assume it's the former as the publication behind the dataset is most likely [this](https://www.nature.com/articles/nature11232). Also [here](https://genetics-docs.opentargets.org/our-approach/data-sources/others). It hink it would be great if we could take a look around the repo and fix everywhere.